ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -842,7 +985,8 @@ void pytensor_module(py::module& m_tensor) {
                 if (py::isinstance<py::list>(tensor)) {
                     return detail::convert_python_tensors_to_tt_tensors(tensor, data_type, tile, strategy);
                 }
-                return detail::convert_python_tensor_to_tt_tensor(tensor, data_type, tile);
+                return detail::convert_python_tensor_to_tt_tensor(
+                    tensor, data_type, Layout::ROW_MAJOR, tile, MemoryConfig{}, nullptr, true);
```

Comment:
Could this result in the user getting a warning if data type is e.g. bfloat8_b?

If yes, is that an acceptable user experience?

Reply:
I think i can switch to just pass `std::nullopt` here since I set the default later. No errors that way for this path.